_practicalities/intro.md

```diff
@@ -284,6 +284,35 @@ Otherwise user code can fail at runtime arbitrarily later in the future, which c
 be a difficult to debug and fix. Get inspiration from the message of failure at
 runtime, and adapt for installation time.
 
+
+Here is for a simple version of how IPython handle old versions of Python and
+Pip that still try to install IPython 7.0+ on Python `< (3,4)`.
+
+```python
+if sys.version_info < (3, 4):
+
+    error = """
+    IPython 7.0+ supports Python 3.4 and above.
+    When using Python 2.7, please install IPython 5.x LTS Long Term Support version.
+    Python 3.3 was supported up to IPython 6.x.
+
+    See IPython `README.rst` file for more information:
+
+    https://github.com/ipython/ipython/blob/master/README.rst
+
+    Python {py} detected.
+
+    Try upgrading pip and retry.
+    """.format(py=sys.version_info)
```

Comment:
The Python version doesn't print correctly for me under Python 2.  Can you check?

I see:

```
Python sys.version_info(major=2, minor=7, micro=14, releaselevel='final', serial=0) detected.
```

Reply:
Perhaps:

```
""".format(py='.'.join([str(v) for v in sys.version_info[:3]]))                           |Python 3, or install an older version of scikit-image using                               

```

Reply:
updated.